server/api/src/rsGeneralAdmin.cpp

```diff
@@ -1135,9 +1138,29 @@ _rsGeneralAdmin( rsComm_t *rsComm, generalAdminInp_t *generalAdminInp ) {
     }
     if ( strcmp( generalAdminInp->arg0, "rm" ) == 0 ) {
         if ( strcmp( generalAdminInp->arg1, "user" ) == 0 ) {
+            const auto user_name = generalAdminInp->arg2 ? generalAdminInp->arg2 : "";
```

Comment:
When using `auto` to deduce a raw pointer, prefer `auto*`.

`auto` by itself is correct/acceptable, but it's not as clear as `auto*`.

_Please apply change to the other lines as well._